tests/integration/config/plugins/integration-tests/data/tests/tests_npc_destination.txt

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2023 by Hurleveur
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+test-data "Npc Destination Save"
+	category "savegame"
+	contents
+		pilot Bobbi Bughunter
+		date 16 11 3013
+		system Sol
+		planet Earth
+		clearance
+		# Set some reputations to positive to avoid combat
+		"reputation with"
+			Bounty 1
+			"Bounty Hunter" 1
+			Pirate 1
+		# What you own:
+		ship "Star Barge"
+			name "Buggy Barge"
+			sprite "ship/star barge"
+			attributes
+				category "Light Freighter"
+				cost 190000
+				mass 70
+				bunks 3
+				"cargo space" 50
+				drag 2.1
+				"engine capacity" 400
+				"fuel capacity" 800
+				"heat dissipation" 0.8
+				hull 1000
+				"outfit space" 1300
+				"required crew" 1
+				shields 600
+				"turret mounts" 1
+				"weapon capacity" 200
+			outfits
+				"X1700 Ion Thruster"
+				"X1200 Ion Steering"
+				Hyperdrive
+				"Anti-Missile Turret"
+				"D14-RN Shield Generator"
+				"nGVF-BB Fuel Cell"
+				"LP036a Battery Pack"
+			crew 1
+			fuel 300
+			shields 600
+			hull 1000
+			engine -9 38 1
+			engine 9 38 1
+			turret 0 -8 "Anti-Missile Turret"
+			leak leak 60 50
+			explode "tiny explosion" 10
+			explode "small explosion" 10
+			system Sol
+			planet Earth
+		account
+			credits 131000
+			score 400
+			history
+		visited Sol
+		"visited planet" Earth
+		visited "Sirius"
+		visited "Procyon"
+		visited "Pollux"
+		visited "Castor"
+
+test-data "Npc Destination Mission"
+	category mission
+	contents
+		mission "npc"
+			landing
+			npc accompany save
+				ship "Quarg Wardragon"
+				government "Quarg"
+				destination Geminus
+				waypoint
```

Comment:
```suggestion
```
Idk if an empty waypoint breaks anything but it shouldnt
We should prob test the mission manually to see what the wardragon does

Reply:
The `waypoint` is necessary. It sets the waypoint to the system of the destination, otherwise it will wander around all over the place until it happens to land on Geminus, and then it won't take off again.

I did test the mission manually, and the wardragon behaved exactly as it was expected to: it went to Geminus and landed, and landing there after it completed the mission.

Specifically, here's the mission as I tested it:
```
mission "Integ Test"
	name "T: Integration Test"
	job
	repeat
	destination Geminus
	npc accompany save
		ship "Quarg Wardragon"
		government "Quarg"
		destination Geminus
		waypoint
		personality uninterested waiting escort target

```

So, I really don't know what's going on here.